applications/ecallmgr/src/ecallmgr_registrar.erl

```diff
@@ -435,6 +437,14 @@ handle_cast({'flush', Realm}, State) ->
 handle_cast({'flush', Username, Realm}, State) ->
     _ = ets:delete(?MODULE, registration_id(Username, Realm)),
     {'noreply', State};
+handle_cast({'gen_listener', {'created_queue', Q}}, State) ->
+    {'noreply', State#state{queue=Q}};
+handle_cast({'gen_listener',{'is_consuming', 'true'}}, #state{queue=Q}=State) ->
+    wapi_registration:publish_sync(
+      [{<<"Queue">>, Q}
```

Comment:
I see. I wonder how we can let future readers know this? Perhaps for now, add a comment in the wapi module?